doc/rfd/rfd/0007/README.md

```diff
@@ -0,0 +1,49 @@
+---
+authors: Jaime Gomes <jaime.gomes@suse.com>
+state: draft
+discussion: 
+---
+
+# RFD 7 Deployment and Upgrade Benchmark
+
+## Introduction
+
+### Motivation
+
+Currently, we do not have any gather data that can corroborate with some eye evidence that
+deployment times of KubeCF are increasing and upgrades are not argument supported in the next
+releases.
+
+### Goal
+
+Be able to measure and collect time metrics during a fresh deployment and between upgrades, on each
+component present on KubeCF.
+
+With the time data collected, both upstream projects and downstream product teams can investigate
+if there're optimizations that can lead to a better and more fasten deployments that will support a
+better user experience in future releases.
+
+### Type of Deployments
+
+1. Eirini enabled:
+1.1 High availability
+2. Diego enabled:
+2.1 High availability
+
+
+## Possible solution(s)
```

Comment:
I would like this RFD to specify the API and kind of data that one receives from the Benchmark, instead of the implementation of it.

Reply:
must contain both at the ends and I am going to add the "kind of data"